tests/algorithms/test_mce_irl.py

```diff
@@ -314,6 +314,31 @@ def test_tabular_policy(rng):
     np.testing.assert_equal(timesteps[0], 2 - mask.astype(int))
 
 
+def test_tabular_policy_rollouts(rng):
+    """Tests tabular policy prediction, especially timestep calculation and masking."""
+    state_space = gym.spaces.Discrete(5)
+    action_space = gym.spaces.Discrete(3)
+
+    s_a = np.stack([np.eye(3)] * 5, axis=1)
```

Comment:
Can we make it clear where `3` and `5` come from? e.g:
```suggestion
    s_a = np.stack([np.eye(action_space.n)] * state_space.n, axis=1)
```
or define `num_states`, `num_actions`